gstd/Cargo.toml

```diff
@@ -27,18 +30,26 @@ futures = { workspace = true, features = ["alloc"] }
 [features]
 #! ## Default features
 
-default = ["panic-handler"]
-## When enabled, a panic handler is provided by this crate.
-panic-handler = []
+default = ["panic-message"]
+
+#! ## Panic handler profiles
+
+## When enabled, a minimal panic handler is provided by this crate.
+## Instead of a panic message, "<unknown>" is displayed.
+panic-handler = ["const_format"]
+## When enabled, a panic handler will also display a panic message.
+panic-message = ["panic-handler", "arrayvec"]
+## When enabled, a panic handler will also display a panic message and location.
```

Comment:
Worth to mention it's not recommended to use in production to avoid sensitive path leak

Reply:
but the stable rust will leak file paths anyway

Reply:
It's anyway good to mention I guess